Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -563,18 +562,29 @@ final class FileManagerTests : XCTestCase {
             File("666", attributes: [.posixPermissions: 0o666])
             File("777", attributes: [.posixPermissions: 0o777])
         }.test {
+            #if os(Windows)
+            // All files are readable on Windows
```

Comment:
Well, there are unreadable files, just that the ACLing properties are far more complex.